src/engine/renderer/tr_image.cpp

```diff
@@ -763,6 +763,14 @@ void R_UploadImage( const byte **dataArray, int numLayers, int numMips, image_t
 
 	int scaledWidth = image->width;
 	int scaledHeight = image->height;
+
+	// If r_imageFitScreen is disabled, use nopicmip instead.
+	if ( ( image->bits & IF_FITSCREEN ) && !r_imageFitScreen.Get() )
+	{
+		image->bits &= ~IF_FITSCREEN;
+		image->bits |= IF_NOPICMIP;
+	}
```

Comment:
Is there a motivation for having this be something we can disable?

Reply:
This is only for asset debugging. We can for example imagine a game that would have a very large 2D minimap larger than screen and that the player could scroll. If someone finds weird the image is blurry, he can disables the feature and deduce that's because the fitscreen flag is set somewhere for that image.

Reply:
The same would be usable for debugging any image that would have received the fitscreen flag but should not use fitscreen like some 3D textures (or worst, lightmaps…).